todo_app/data/Trello.py

```diff
@@ -128,6 +125,8 @@ def update_to_done(cls, item_id):
     @staticmethod
     def create_board(board_name):
         params = trello.headers | {ResponseFilter.name: board_name}
+        print(params)
+        print(trello.boards_url)
```

Comment:
We should delete these, particularly because they're printing your parameters including potentially sensitive keys!